src/main/java/edu/utdallas/davisbase/host/Host.java

```diff
@@ -39,9 +42,68 @@ public Host(HostConfiguration configuration, BufferedReader reader, PrintWriter
     this.writer = writer;
   }
 
-  public String readStatement() throws IOException {
-    // TODO Implement Host.readStatement()
-    throw new NotImplementedException();
+  static String prompt = "davisql> ";
```

Comment:
Make this a `final` _instance_ field of `HostConfiguration`. Let it be set in the constructor. You may default it though if not explicit value is given.